components/pedigree/resources/src/main/resources/pedigree/view/nodeMenu.js

```diff
@@ -757,19 +757,23 @@ define([
                         });
                     }
 
-                    var processLinking = function(topMessage, notesMessage) {
-                        editor.getOkCancelDialogue().showWithCheckbox("<br><b>" + topMessage + "</b><br>" +
-                            "<div style='margin-left: 30px; margin-right: 30px; text-align: left'>Please note that:<br><br>"+
-                            notesMessage + "</div>",
-                            "Add patient to the family?",
-                            "Do not show this warning again<br>", false,
-                            "Confirm", function(checkBoxStatus) { setDoNotShow(checkBoxStatus); processCreatePatient() },
-                            "Cancel",  function(checkBoxStatus) { setDoNotShow(checkBoxStatus); });
-                    }
+                    if (!editor.getPreferencesManager().getConfigurationOption("hideShareConsentDialog")) {
+                        var processLinking = function(topMessage, notesMessage) {
+                            editor.getOkCancelDialogue().showWithCheckbox("<br><b>" + topMessage + "</b><br>" +
```

Comment:
I will, but what is the practical difference (since those strings are dynamically generated in JS no static HTML document on disc is wrong, and dynamically all browsers show both variants correctly)?